ComponentSnapshotTestCase/CKComponentSnapshotTestCase.mm

```diff
@@ -8,7 +8,7 @@
  *
  */
 
-#import <ComponentSnapshotTestCase/CKComponentSnapshotTestCase.h>
+#import <ComponentKit/CKComponentSnapshotTestCase.h>
```

Comment:
Hm, this won't work with our internal build process (which isn't cocoapods based).

How is it that `CKComponentSnapshotTestCase` lives in ComponentKit proper, as opposed to a separate sub-library? We specifically want to avoid linking it into production builds.

Can you explain this a little more? I don't know a lot about Cocoapods.